media/js/src/utils.js

```diff
@@ -326,6 +326,10 @@ const getAssetType = function(primaryType) {
         type = 'video';
     } else if (primaryType === 'image_fpxid' || primaryType === 'image_fpx') {
         type = 'image';
+    } else if (primaryType === 'mp4_audio') {
+        type = 'audio';
+    } else {
+        type = 'unknown';
```

Comment:
`primary_type` can also === 'image' here. For reference, all the labels are listed here: https://github.com/ccnmtl/mediathread/blob/master/mediathread/assetmgr/models.py#L154